### ISSUE TITLE
ECM Field: Shouldn't it stay around you once launched when you are moving?

### ISSUE THREAD

**Issue by Bertram25** [stephenjsweeney/tbftss#8]
When launching a ECM Field, the field stays around you, either you are moving or not.
This felt a bit unnatural to me, but maybe it is desired.

Typically, it would make it hard to ECM the lieutenant pirate if it stays where it was triggered, but in other cases, it could practical to trigger it while being chased.

Cheers,

**Comment by stephenjsweeney** [comment]
The ECM is used as defense against missiles targeting the player. Its effects last 1 second, and will destroy any missiles tracking the player at that moment.

In order to capture the pirate lieutenant (Pirate Uprising #3), you need to select the Ray's mag cannons (Z or mouse wheel up), and fire those at his craft.

**Comment by Bertram25** [comment]
> The ECM is used as defense against missiles targeting the player. Its effects last 1 second, and will destroy any missiles tracking the player at that moment.

Sorry for the confusion about the use of ECMs. I indeed used the mag cannon for the Pirate Lieutenant last time I tried.

Even though, I wondered whether ECM emission should stick around the ship once triggered, even if it is moving, or not. To me, it looked more realistic when the wave would not be relative to the ship, but to the point where it was triggered at. (Again, this is just a suggestion, as I know that when developing a game, we sometimes stop seeing even obvious bugs, if you consider this as being one.)

Cheers, :)
